UI/CombatReport/GraphicalSummary.cpp

```diff
@@ -352,18 +352,18 @@ class SideBar : public GG::Wnd {
 
         GG::Clr axis_label_color = combat_summary.SideColor();
 
-        m_x_axis_label = new CUILabel( (boost::format( UserString("COMBAT_FLEET_HEALTH_AXIS_LABEL") )
+        m_x_axis_label = GG::Wnd::Create<CUILabel>( (boost::format( UserString("COMBAT_FLEET_HEALTH_AXIS_LABEL") )
                                       % combat_summary.SideName()
```

Comment:
odd indenting. could just revert to a single tab (4 spaces) indent from the previous line